airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1533,6 +1532,7 @@ definitions:
           - type: string
         interpolation_context:
           - config
+          - response
```

Comment:
@cgkournelos-rf what is the CDK change being made here? could you split it out into its own PR?

Reply:
This is not something that we changed. It seems that it was brought with the last rebase onto the master. We will fix it!

Reply:
Fixed